data/mercator.py

```diff
@@ -1,60 +1,64 @@
-import math
-import re
-import warnings
+from typing import Union
 
-import cftime
 import numpy as np
 import pyresample
-import pytz
-from cachetools import TTLCache
-from netCDF4 import Dataset
 from pint import UnitRegistry
 
+import data.geo as geo
 from data.calculated import CalculatedData
+from data.model import Model
 from data.nearest_grid_point import find_nearest_grid_point
-from data.variable import Variable
-from data.variable_list import VariableList
+from data.netcdf_data import NetCDFData
 from utils.errors import APIError
 
 
-class Mercator(CalculatedData):
+class Mercator(Model):
     __depths = None
 
-    def __init__(self, url, **kwargs):
+    def __init__(self, nc_data: Union[CalculatedData, NetCDFData]) -> None:
```

Comment:
Since `__init__` doesn't explicitly return anything (https://help.semmle.com/wiki/display/PYTHON/__init__+method+returns+a+value), does it make sense to annotate the return type?

Reply:
I wondered about that, but [PEP-484 says explicitly](https://www.python.org/dev/peps/pep-0484/#id13) that "the return type of `__init__` ought to be annotated with `-> None`" and provides a rationale.